src/components/banner/Banner.module.scss

```
@@ -0,0 +1,103 @@
+@use 'theme';
+
+$container-align-items: var(--container-align-items);
+$container-grid-template-columns: var(--container-grid-template-columns);
+$header-color: var(--header-color);
+
+.styledBanner {
+  border-radius: theme.$amino-radius-12;
+  padding: theme.$amino-space-16;
+  [class~='amino-button'] {
+    & :hover,
+    &:focus {
+      filter: brightness(1.1);
+    }
+  }
```

Why do you need to select for the class that has a relatively similar `amino-button` className? Would `.amino-button` not work? 🤔 
```suggestion
 .amino-button {
    & :hover,
    &:focus {
      filter: brightness(1.1);
    }
  }
```

---

unfortunately, that doesn't work. If Sass/css-modules sees a normal class in the module like that, it converts it to random string (ex. OL8_98nAGpDA6aY2B6A). It does this differently in each module allowing them to be scoped. So this random string will be different than the class that is on the button we are trying to style.
I've done some looking around, and it doesn't look like there is a great way to target classes from other modules like this. That's the whole point of the css-module philosophy, I guess. There are suggestions that we could use a `:global` class, but it's not ideal.

---

EDIT: Thien and I talked and this and decided that the best way to access the `amino-button` class on the Button is to use `:global(.amino-button) {}`. This prevents post-css from transforming the class name, leaving it as normal css. We will use this syntax any time we need to access a class from another module.